.github/workflows/ci.yml

```diff
@@ -66,6 +66,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1
+        with:
+          key: windows-${{ matrix.platform }}
```

Comment:
Other platforms use 
```suggestion
          key: ${{ matrix.cache_key }}
```

Reply:
I know, but then they have this to define the cache_key, for example:
```
        platform: [win32]
        include:
          - platform: win32
            platform_name: win32
            cache_key: windows-mingw
            build_flags: -DBUILD_SHARED_LIBS=ON -DENABLE_SCRIPTING=ON
```

Since I would be branching out windows platforms just to define the `cache_key` which would then be the same as it currently is, I decided to go the simpler route.